Makefile

```diff
@@ -82,18 +82,21 @@ ifdef GOBIN
 TANZU_CLI_INSTALL_PATH = "$${GOBIN}/tanzu"
 endif
 
+NUMBER_OF_CORES ?= 1
 ifeq ($(TCE_CI_BUILD), true)
 XDG_DATA_HOME := /tmp/mylocal
 XDG_CACHE_HOME := /tmp/mycache
 XDG_CONFIG_HOME := /tmp/myconfig
 SED := sed -i
+NUMBER_OF_CORES := $(shell nproc --all)
```

Comment:
Wouldn't we hit the linux check on line 99? Harmless to do it twice, but also duplication.

Reply:
No, we won't. This first part is to satisfy the build in a container. So if `TCE_CI_BUILD=true` everything else after that is skipped because XDG_DATA_HOME is set. It was basically to treat the CI build (which is a linux OS) different from a native Linux system like on a laptop. Lines 102 to 110 wont execute.